flutter_appauth/android/src/main/java/io/crossingthestreams/flutterappauth/FlutterAppauthPlugin.java

```diff
@@ -307,6 +307,11 @@ private void performAuthorization(AuthorizationServiceConfiguration serviceConfi
             authRequestBuilder.setResponseMode(responseMode);
         }
 
+        if(additionalParameters != null && !additionalParameters.isEmpty() && additionalParameters.containsKey("ui_locales")){
```

Comment:
The first two conditions of this `if` statement are duplicated in the next `if` statement that existed prior to this PR. Would you be able to refactor the code to remove the duplication?

Reply:
Sure i ll change it

Reply:
I pushed one more commit with changes.